src/app/modules/item/http-services/get-item-by-id.service.ts

```diff
@@ -8,6 +8,7 @@ export interface Item {
   requires_explicit_entry: boolean
   string: {
     title: string|null,
+    subtitle: string|null,
```

Comment:
Subtitle is both nullable and omitable (cfr API, "nullable" and not required), so it is `subtitle?: string|null`

Reply:
fixed